tests/standAloneClient/specificCategoryEntryNotification.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xml>
+	<config>
+		<serviceUrl>http://{prompt:Host name:}/</serviceUrl>
+		<partnerId>-2</partnerId>
+		<clientTag>Stand alone php 1.0.0</clientTag>
+		<curlTimeout>30</curlTimeout>
+		<userAgent>Stand alone php 1.0.0</userAgent>
+	</config>
+	<session>
+		<!-- script will ask for username / password interactively -->
+	</session>
+	<multirequest>
+		<!-- New Item Pending Moderation -->
+		<request service="eventNotificationTemplate" action="add" plugin="eventNotification" partnerId="1968271">
+			<template objectType="KalturaEmailNotificationTemplate">
+				<name>Entry Published to Specific Category</name>
+				<systemName>ENTRY_PUBLISHED_SPECIFIC_CATEGORY</systemName>
+				<description>Email notification template to be sent when an entry is published to specific category</description>
+				<automaticDispatchEnabled>1</automaticDispatchEnabled>
+				<eventType>5</eventType> <!-- EventNotificationEventType::OBJECT_CREATED -->
+				<eventObjectType>37</eventObjectType> <!-- EventNotificationEventObjectType::CATEGORYENTRY -->
+				<eventConditions objectType="array">
+					<item objectType="KalturaFieldMatchCondition">
+						<description>Match category ID</description>
+						<field objectType="KalturaEvalStringField">
+							<code>$scope->getEvent()->getObject()->getCategoryId()</code>
```

Comment:
Please use $scope->getObject()->getCategoryId() in order to support manual dispatch as well.